tests/sriov_test.go

```diff
@@ -23,32 +23,37 @@ import (
 	"fmt"
 	"io/ioutil"
 	"testing"
+	"time"
+
+	"github.com/skydive-project/skydive/common"
 )
 
+var device = "enp5s0f0"
```

Comment:
It seems we need to detect NIC being UP and supporting SRIO-V otherwise the test will fail

Reply:
implemented as findSRIOV as discussed on irc: check for sriov_numvfs as a proof it is an sriov interface and check for operstate == up.